_includes/templates/download.html

```diff
@@ -90,8 +90,10 @@ <h2>{{ page.latestversion }} {{CURRENT_RELEASE}} <a type="application/rss+xml" h
         <span>
           <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64le }}.tar.gz" class="dl">PPC64 Linux</a>
             <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64 }}.tar.gz" class="dl" id="lin64ppc">64 bit</a>
+<!--
             -
             <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64le }}.tar.gz" class="dl" id="lin64ppcle">64 bit LE</a></span>
+-->
```

Comment:
Dangling reference to `ppc64le` here is breaking CI. Since there's no longer the choice between two options, I would just update the "header" to `ppc64` and remove the two "options" altogether, instead of just the `ppc64le` one.
```suggestion
          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64 }}.tar.gz" class="dl">PPC64 Linux</a>
<!--
            <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64 }}.tar.gz" class="dl" id="lin64ppc">64 bit</a>
            -
            <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64le }}.tar.gz" class="dl" id="lin64ppcle">64 bit LE</a></span>
-->
```

Reply:
Done